GNUmakefile

```diff
@@ -798,32 +801,36 @@ ifneq ($(only_unit),1)
     endif
 endif
 
-install:
-	cd blaspp   && $(MAKE) install prefix=${prefix}
+install: lib ${pkg}
+	cd blaspp   && ${MAKE} install prefix=${abs_prefix}
 	@echo
-	cd lapackpp && $(MAKE) install prefix=${prefix}
+	cd lapackpp && ${MAKE} install prefix=${abs_prefix}
 	@echo
-	mkdir -p $(DESTDIR)$(prefix)/include/slate/internal
-	mkdir -p $(DESTDIR)$(prefix)/lib$(LIB_SUFFIX)
-	cp include/slate/*.hh          $(DESTDIR)$(prefix)/include/slate
-	cp include/slate/internal/*.hh $(DESTDIR)$(prefix)/include/slate/internal
-	cp lib/lib*                    $(DESTDIR)$(prefix)/lib$(LIB_SUFFIX)
-	if [ $(c_api) -eq 1 ]; then \
-		mkdir -p $(DESTDIR)$(prefix)/include/slate/c_api; \
-		cp include/slate/c_api/*.h  $(DESTDIR)$(prefix)/include/slate/c_api; \
-		cp include/slate/c_api/*.hh $(DESTDIR)$(prefix)/include/slate/c_api; \
+	mkdir -p ${DESTDIR}${abs_prefix}/include/slate/internal
+	mkdir -p ${DESTDIR}${abs_prefix}/lib${LIB_SUFFIX}
+	mkdir -p ${DESTDIR}${abs_prefix}/lib${LIB_SUFFIX}/pkgconfig
+	cp include/slate/*.hh          ${DESTDIR}${abs_prefix}/include/slate/
+	cp include/slate/internal/*.hh ${DESTDIR}${abs_prefix}/include/slate/internal/
+	cp lib/lib*                    ${DESTDIR}${abs_prefix}/lib${LIB_SUFFIX}
+	cp ${pkg}                      ${DESTDIR}${abs_prefix}/lib${LIB_SUFFIX}/pkgconfig/
+	if [ ${c_api} -eq 1 ]; then \
+		mkdir -p ${DESTDIR}${abs_prefix}/include/slate/c_api; \
+		cp include/slate/c_api/*.h  ${DESTDIR}${abs_prefix}/include/slate/c_api; \
+		cp include/slate/c_api/*.hh ${DESTDIR}${abs_prefix}/include/slate/c_api; \
```

Comment:
Is it normal to use `abs_prefix` with `DESTDIR`? To me, when I read it, it seems to prepend the absolute path of the prefix with `DEST_DIR`.

Reply:
Yes. DESTDIR is to stage something in a tmp place that looks exactly like the regular filesystem. We don't use it, but it's there per GNU coding standards.
https://www.gnu.org/prep/standards/html_node/DESTDIR.html